pkg/apis/broker/v1beta1/trigger_lifecycle.go

```diff
@@ -65,11 +65,11 @@ func (ts *TriggerStatus) PropagateBrokerStatus(bs *BrokerStatus) {
 
 	switch {
 	case bc.Status == corev1.ConditionUnknown:
-		ts.MarkBrokerUnknown(bc.Reason, bc.Message)
+		ts.MarkBrokerUnknown("Broker-"+bc.Reason, bc.Message)
```

Comment:
Optional nit: I think I'd prefer `/` as separator here. That seems to be the preferred condition name separator for Pod Lifecycle according to https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-readiness-gate.